R/nicenumericcuts.R

```diff
@@ -388,14 +372,13 @@ NiceNumericCuts <- function(input.data,
                 cuts <- seq(start, end, length.out = num.categories + 1)    
             } else {
                 cuts <- seq(start, end, by = equal.intervals.increment)
-                if (cuts[length(cuts)] != end) {
-                    warning("The range of values cannot be evenly divided by the increment of ", 
-                        equal.intervals.increment, ". Please check the value of the Increment ",
-                        "setting.")
+                if (cuts[length(cuts)] < end) {
+                    cuts = c(cuts, cuts[length(cuts)] + equal.intervals.increment)
```

Comment:
Could omit this by just using the `length.out` argument to force it to add an extra element if it doesn't reach the boundary
e.g. 
```suggestion
                cuts <- seq(from = start, by = equal.intervals.increment, length.out = ceil((end - start)/equal.intervals.increment))
```